### DISCUSSION TITLE
Sometimes when loading into a save and completing the ongoing mission the campaign map wont reveal

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#14235] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/55637865/aa8dd306-f163-4e01-bf31-db1a56728798)
arrived at the natural formation, map didn't further reveal so i couldn't progress further

### Reproduction steps

find a spot where a natural formation is at the edge of the revealed sector of the map
start a mission heading towards that formation
leave the campaign
go back in
when you get to the formation see if the map loads or not

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.100.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I unfortunately couldn't find a spot where this'd occur. Here's an example campaign save, where the destination seems to be covered by the fog of war the same way as in your screenshot: 
[Save 38.zip](https://github.com/user-attachments/files/16414436/Save.38.zip)

The FoW does seem to get correctly cleared when I reach the end of the level. So I suspect this might require some more specific map layout or situation to occur. If you have a campaign save or a seed available where this occurs consistently, that'd help us a lot in diagnosing the issue. Thanks and sorry for the trouble!
